site/docs/components/TokenSelectorContainer.tsx

```diff
@@ -0,0 +1,12 @@
+import { ReactElement, useState } from 'react';
+import type { Token } from '@coinbase/onchainkit/token';
+
+type TokenSelectorContainer = {
```

Comment:
btw let's keep have all the types in the types.ts file, even the one we don't export.